athena/entrypoints/download.py

```diff
@@ -0,0 +1,58 @@
+import datetime
+import tempfile
+from pathlib import Path
+
+
+from athena.apicultor.client import BinanceClient
+from athena.apicultor.fetch import fetch_historical_data
+from athena.core.interfaces import Fluctuations, DatasetLayout
+from athena.core.types import Coin, Period
+from tqdm import tqdm
+
+
+# TODO : CLI asking for coin / currency / start_date / end_date / period / ...
+
+
+def download_market_candles(
+    coin: Coin,
+    currency: Coin,
+    from_date: datetime.datetime,
+    to_date: datetime.datetime,
+    period: Period,
+    output_dir: Path,
+) -> None:
+    """Download market data from coin / currency pair and save it.
```

Comment:
```suggestion
    """Download market data from coin / currency pair as fluctuations and save them.
```
Would be great to find a better name for fluctuations.